### DISCUSSION TITLE
Timestamps - what am I missing?

### DISCUSSION THREAD

**Discussion by keresian** [aaronwmorris/indi-allsky#1608] [Q&A]
I'm trying to set up File Transfer (via ftp if it matters, on a LAN, so encryption doesnt matter).  Despite me trying to set the remote to image-{ts}.{ext}, what gets saved is a random string.jpg.  How do I get it to actually use the timestamp?

**Comment by aaronwmorris** [comment]
The `ts` and `timestamp` variables are datetime variables.  There is not a strftime() variable to get the unix timestamp string.

Because `ts` is a datetime object, you have to add the strftime() formats for the date.

You could use something like `image-{ts:%Y%m%d_%H%M%S}.{ext}`

**Comment by keresian** [comment]
Thanks Aaron, this along with some investigation figured out my issue.   The timestamp was setting fine, but I was viewing the FTP'd files via a SMB share, and apparently it didnt like : and such in the filename.  After I reformatted the name to be YYYYMMDD-HH_MM_SS, it started displaying the values correctly via the network share.

Thanks for the quick assist.
